markdown/setup/aqua-data-studio-setup.md

```diff
@@ -0,0 +1,43 @@
+### Aqua Data Studio
+[Link to product webpage](https://www.aquafold.com/aquadatastudio/).
+
+#### Using Aqua Studio with Generic JDBC driver
+1. Locate the Server in the Server Browser
```

Comment:
Where is the Server Browser? It is better to have a picture to know how to operate.

Reply:
Good point. I have added the picture and refactored the documentation